update.sh

```diff
@@ -8,44 +8,52 @@
 
 set -e
 
+# Check whether there is a newer version of this script
+cd /usr/share/rhino/rhino-update
+if git pull; then
+    chmod +x /usr/share/rhino/rhino-update/update.sh
+    exec /usr/bin/rhino-update
+    exit $?
+fi
+
 # Check to see whether the "configuration update", released in 2022.04.19 has been applied.
-if [[ ! -f "$HOME/.rhino/updates/configuration" ]]; then
-  mkdir -p ~/.rhino/{config,updates}
-  echo "alias rhino-config='mkdir ~/.rhino/config/config-script && git clone https://github.com/rollingrhinoremix/rhino-config ~/.rhino/config/config-script/ && python3 ~/.rhino/config/config-script/config.py && rm -rf ~/.rhino/config/config-script'" >> ~/.bashrc
-  : > "$HOME/.rhino/updates/configuration"
+if [[ ! -f "/usr/share/rhino/updates/configuration" ]]; then
+  mkdir -p /usr/share/rhino/{config,updates}
+  echo "alias rhino-config='mkdir /usr/share/rhino/config/config-script && git clone https://github.com/rollingrhinoremix/rhino-config /usr/share/rhino/config/config-script/ && python3 /usr/share/rhino/config/config-script/config.py && rm -rf /usr/share/rhino/config/config-script'" >> ~/.bashrc
+  : > "/usr/share/rhino/updates/configuration"
 fi
 
 # Check to see whether the rhino-config v2 update has been applied, which converts Rhino into a command-line utility.
-if [[ ! -f "$HOME/.rhino/updates/config-v2" ]]; then
-  mkdir ~/rhinoupdate/distro
-  git clone https://github.com/rollingrhinoremix/distro ~/rhinoupdate/distro
-  mv ~/rhinoupdate/distro/.{bashrc,bash_aliases} ~
-  : > "$HOME/.rhino/updates/config-v2"
+if [[ ! -f "/usr/share/rhino/updates/config-v2" ]]; then
+  mkdir /usr/share/rhino/rhinoupdate/distro
+  git clone https://github.com/rollingrhinoremix/distro /usr/share/rhino/rhinoupdate/distro
+  mv /usr/share/rhino/rhinoupdate/distro/.{bashrc,bash_aliases} ~
+  : > "/usr/share/rhino/updates/config-v2"
 fi
```

Comment:
All of this could be removed as it's un-necessary for the new version, this would be the rhino-update v2 command and does not require the use of bash aliases that this does.